qa/workunits/rbd/concurrent.sh

```diff
@@ -88,11 +93,8 @@ function setup() {
 
 	# Set up some environment for normal teuthology test setup.
 	# This really should not be necessary but I found it was.
-	TOP="/tmp/cephtest"
-	export CEPH_ARGS="--conf ${TOP}/ceph.conf"
-	export CEPH_ARGS="${CEPH_ARGS} --keyring ${TOP}/data/client.0.keyring"
-	export CEPH_ARGS="${CEPH_ARGS} --name client.0"
 
+	export CEPH_ARGS=" --name client.0"
 	export LD_LIBRARY_PATH="${TOP}/binary/usr/local/lib:${LD_LIBRARY_PATH}"
```

Comment:
don't need these LD_LIBRARY_PATH or PATH settings anymore - especially now that TOP isn't defined